src/util/BrowserFeatures.js

```diff
@@ -55,7 +55,7 @@ define(function () {
   };
 
   fn.isIE = function () {
-    return /msie/i.test(navigator.userAgent);
+    return /msie/i.test(navigator.userAgent) || /trident/i.test(navigator.userAgent);
```

Comment:
Is this how we want to check for IE at all? Why not see if document.documentMode exists?